packages/core/content-releases/admin/src/pages/Releases.tsx

```diff
@@ -1,22 +1,56 @@
-import { Main } from '@strapi/design-system';
-import { Route, Switch } from 'react-router-dom';
+/*
+ *
+ * Releases
+ *
+ */
+import React from 'react';
 
-import { pluginId } from '../pluginId';
+import { Button, Main, HeaderLayout } from '@strapi/design-system';
+import { Plus } from '@strapi/icons';
+import { useIntl } from 'react-intl';
+
+import { AddReleaseDialog } from '../components/AddReleaseDialog';
+
+const ReleasesPage = () => {
+  const [showAddReleaseDialog, setShowAddReleaseDialog] = React.useState(false);
```

Comment:
showAddReleaseDialog sounds like a function. For booleans we try to stick with names with "is" or "has". In this case I'd recommend `addReleaseDialogIsShown`. It's a bit longer but worth it for clarity in my opinion